Exiled.CustomRoles/API/Features/KeypressActivator.cs

```diff
@@ -95,7 +95,20 @@ private IEnumerator<float> ProcessAltKey(Player player)
                 _ => AbilityKeypressTriggerType.None,
             };
 
-            player.ShowHint(!PreformAction(player, type, out string response) ? $"Failed to preform action: {response}" : $"Preformed action: {response}", 5f);
+            bool preformed = PreformAction(player, type, out string response);
+            switch (preformed)
+            {
+                case true when type == AbilityKeypressTriggerType.Activate:
+                    string[] split = response.Split('|');
+                    response = string.Format(CustomRoles.Instance.Config.UsedAbilityHint.Content, split);
+                    break;
+                case false:
+                    response = string.Format(CustomRoles.Instance.Config.FailedActionHint.Content, response);
+                    break;
+            }
```

Comment:
The use of a switch here is to reduce branching if more cases are added in the future. For example, if another action is added that have a differently formated string, for example of you want to `case true when type == Description`, it will use a hashtable for the switch statement instead of a simple branch. The hashtable would be faster the more cases there are, while in this simple form there's no impact either way.